### DISCUSSION TITLE
Is the ddraw and d3d limition on apple silicone still true?

### DISCUSSION THREAD

**Discussion by Kethen** [The-Wineskin-Project/WineskinServer#242] [Q&A] [answered]
First of all, thanks for the effort in maintaining and building wine for mac

On the release page:

> Apple Silicon Support;
> 
> This will now function under Rosetta2 use WineCX19 or greater, direct draw & Direct3D won't function so things like cnc-ddraw & DxWnd can be used to workaround this limitation.

Doing a google search points towards https://github.com/Gcenx/WineskinServer/issues/65 but the issue page was removed from this project

If it is still an issue, does it only affect opengl wined3d, aka the unfortunate 32bit directx9 games that won't work with dxvk?

**Comment by Gcenx** [comment, marked as answer]
All Apple silicon systems have this issue but this only affects older title that use lower screen resolutions that these systems don’t advertise.

CodeWeavers did hack a fshack to workaround this problem but it’s not completely resolved the problem, using CX20.0.4 and later and connecting a monitor to the HDMI port there should be no issue.

Some of the newer MacBooks have this problem where some don’t, again connecting a monitor there’s no issue.

As that’s not always possible or wanted using cnc-ddraw/DxWnd will work around the problem I’ve tested both of these even before CodeWeavers implemented there little hack.

Correct you can’t use DXVK for anything other than 64Bit DirectX 10/11 games due to limitations in the Metal API.
